lib/elixir/test/elixir/file_test.exs

```diff
@@ -844,7 +864,40 @@ defmodule FileTest do
 
     test "open respects encoding" do
       {:ok, file} = File.open(fixture_path("utf8.txt"), [{:encoding, :latin1}])
-      assert IO.gets(file, "") == <<195, 144, 194, 160, 195, 145, 194, 131, 195, 145, 194, 129, 195, 145, 194, 129, 195, 144, 194, 186, 195, 144, 194, 184, 195, 144, 194, 185, 10>>
+
+      data = <<
```

Comment:
Not sure if there is a better way to format this binary or if this is the preferred way based on the formatter.

Reply:
Probably break it apart using string concatenation?

```
<<1, 2, 3, 4, 5, 6>> <>
  <<1, 2, 3, 4, 5, 6>>
```

?